tests/test_dependencies.py

```diff
@@ -55,3 +65,43 @@ def test_modules() -> None:
     )
 
     assert error_code == 0, stderr
+
+
+@pytest.mark.skipif(not deps, reason="galaxy-lib is not installed")
+@pytest.mark.skipif(
+    not spawn.find_executable("modulecmd"), reason="modulecmd not installed"
+)
+def test_modules_environment(tmp_path: Path, monkeypatch: Any) -> None:
+    """Here check that the environment variables set by a module are
+    being propagated into a tool's environment, by running `env` as the
+    tool and parsing its output.
+    """
```

Comment:
```suggestion
    """
    Check that the environment variables set by a module are being propagated correctly.

    Do so by  by running `env` as the tool and parsing its output.
    """
```

To fix https://travis-ci.com/github/common-workflow-language/cwltool/jobs/454295930#L4191